cmake/FindPAPI.cmake

```diff
@@ -41,11 +45,11 @@ if (UNIX)
 
     # Make PAPI an imported target and also include its public header papi.h
     if (PAPI_FOUND AND NOT TARGET PAPI::PAPI)
-        add_library(PAPI::PAPI SHARED IMPORTED)
+        add_library(PAPI::PAPI STATIC IMPORTED)
         set_target_properties(PAPI::PAPI PROPERTIES IMPORTED_LOCATION ${PAPI_LIBRARY})
-        target_include_directories(PAPI::PAPI INTERFACE ${PAPI_INCLUDE_DIR})
```

Comment:
I think `set_target_properties(PAPI::PAPI PROPERTIES IMPORTED_LOCATION ${PAPI_LIBRARY} INTERFACE_INCLUDE_DIRECTORIES ${PAPI_INCLUDE_DIR})` should also work and will also limit the scope of `${PAPI_INCLUDE_DIR}`